docfx-util.ps1

```diff
@@ -0,0 +1,57 @@
+
+[CmdletBinding(PositionalBinding=$false)]
+param 
+(
+    [parameter(mandatory=$false)][switch][Alias('c')]$clean,
+    [parameter(mandatory=$false)][switch][Alias('d')]$doclinkchecker,
+    [parameter(mandatory=$false)][string][Alias('l')]$lychee,
+    [parameter(mandatory=$false)][string][Alias('a')]$all
+)
+
+
+
+# this is called removeartifacts instead of clean because clean might be taken up by another name
+function removeartifacts
+{
+    $deletePaths = '.\workflows\*.svg', '.\api\*.yml', '.\api\.manifest', '.\_site\', '.\_raw\', '.\_view\'
```

Comment:
This did not actually remove the existing SVGs from my machine, I had to modify the first value to `'.\workflows\**\*.svg'` so that it would recurse correctly through and remove the SVG files